spec/models/population_spec.rb

```diff
@@ -0,0 +1,176 @@
+require 'rails_helper'
+
+RSpec.describe Population do
+  it { is_expected.to belong_to(:location) }
+
+  it { is_expected.to validate_presence_of(:date) }
+  it { is_expected.to validate_presence_of(:operational_capacity) }
+  it { is_expected.to validate_presence_of(:usable_capacity) }
+  it { is_expected.to validate_presence_of(:unlock) }
+  it { is_expected.to validate_presence_of(:bedwatch) }
+  it { is_expected.to validate_presence_of(:overnights_in) }
+  it { is_expected.to validate_presence_of(:overnights_out) }
+  it { is_expected.to validate_presence_of(:out_of_area_courts) }
+  it { is_expected.to validate_presence_of(:discharges) }
+  it { is_expected.to validate_numericality_of(:operational_capacity) }
+  it { is_expected.to validate_numericality_of(:usable_capacity) }
+  it { is_expected.to validate_numericality_of(:unlock) }
+  it { is_expected.to validate_numericality_of(:bedwatch) }
+  it { is_expected.to validate_numericality_of(:overnights_in) }
+  it { is_expected.to validate_numericality_of(:overnights_out) }
+  it { is_expected.to validate_numericality_of(:out_of_area_courts) }
+  it { is_expected.to validate_numericality_of(:discharges) }
+
+  describe '#moves_from' do
```

Comment:
Think we can remove this describe block now we don't use this method?

Reply:
It's used